### ISSUE TITLE
Percolator wrapper crashing when enzyme is set to nonspecific

### ISSUE THREAD

**Issue by MKoesters** [ursgal/ursgal#73]
When setting the value of the parameter enzyme to nonspecific, the following line in the percolator wrapper crashes:
cleavage_site = self.params['translations']['enzyme'].split(';')[1]

Since there is no percolator2 translation for enzyme "nonspecific", its not changed to the internal format and there is no list entry 1 after splitting

I suggest, to always set enzN and enzC in the mascot wrapper always to 1 (True) and misscleavages to zero when the enzyme is nonspecific, however im not 100% sure about this.

**Comment by StSchulze** [comment]
For percolator, there needs to be a translation in uparams['enzyme'] for any enzyme that you want to use. So if you want to use nonspecific, just add it to the translation. I'm not sure right now, how percolator uses it, but for msamanda it's e.g. just ';;' to allow the splitting.

The part about the mascot wrapper I don't understand, but I think in general, the cleavage site should never be C- and N-terminal of the specified amino acid but only one of the two, for nonspecific it doesn't matter, since it can cleave before/after any amino acid anyway.

**Comment by MKoesters** [comment]
Sry, should also be percolator wrapper ...
Sry, maybe I was a bit unclear
The variables enzN and enzC specify, if the cleavage is how it should be, according to the specified enzyme
If the enzyme is nonspecific, all cleavages are "correct", since the enzyme can cut anywhere
Also, you cant really figure out how many miscleavages you have (which is a feature for the percolator input) since there is no definition, where the enzyme should cut

**Comment by StSchulze** [comment]
Oh, I see, then you are right (sry, I guess I could have looked into the wrapper as well ...),  enzN and enzC are always correct and missed cleavages 0.

**Comment by MKoesters** [comment]
I'll do a pull request tomorrow, maybe we can shortly discuss this in the dev meeting :)

**Comment by StSchulze** [comment]
Since this came up again in a different context (#94), I have included the nonspecific uvalue_translation for percolator in uparams now. I have used "ACDEFGHIKLMNPQRSTVWY;C;", which works fine for "enzN" and "enzC" without any changes in the wrapper. However, missed cleavages will be basically the length of the peptide (-1). Same for unify_csv.
I'm not sure if it should rather be 0, as mentioned before!? At the moment both versions seem to make sense to me, so let's discuss that?
